precompile/contracts/rewardmanager/contract_test.go

```diff
@@ -93,6 +96,118 @@ var (
 				require.False(t, isFeeRecipients)
 			},
 		},
+		"set allow fee recipients from manager succeeds before activation": {
+			Caller:      allowlist.TestManagerAddr,
+			ChainConfig: precompileconfig.NewMockChainConfig(commontype.ValidTestFeeConfig, false, nil),
+			BeforeHook:  allowlist.SetDefaultRoles(Module.Address),
+			InputFn: func(t testing.TB) []byte {
+				input, err := PackAllowFeeRecipients()
+				require.NoError(t, err)
+
+				return input
+			},
+			SuppliedGas: AllowFeeRecipientsGasCost,
+			ReadOnly:    false,
+			ExpectedRes: []byte{},
+			AfterHook: func(t testing.TB, state contract.StateDB) {
+				_, isFeeRecipients := GetStoredRewardAddress(state)
+				require.True(t, isFeeRecipients)
+			},
+		},
```

Comment:
just to confirm,  this should be an impossible case to hit because we ensure that a manager will not be present in the state prior to DUpgrade activation?

Reply:
true. removed those before/after tests as they behave same.